tulip/gridworld.py

```diff
@@ -1256,13 +1258,13 @@ def update_line(num, dlist, lines):
         lines.append((l, l_trail))
 
     if not save_prefix:
-        ani = matplotlib.animation.FuncAnimation(fig, update_line,
-                                                 len(paths[0]),
-                                                 fargs=(data, lines),
-                                                 interval=500)
-        plt.show()
```

Comment:
Deleting `plt.show()` here is significant because, after this change, the user must call `matplotlib.pyplot.show()` (or similar) to display the animation.

I think this is a good change, but I will add a comment to the docstring to help users who might expect the animation to immediately appear.